arch/riscv/mach-dpu/ssi_flash.c

```diff
@@ -49,27 +49,37 @@ void __dpu_ssi_flash_boot(void *boot, uint32_t addr, uint32_t size)
 {
 	int i;
 	uint8_t *dst = boot;
-	void (*boot_entry)(void) = boot;
 
-	for (i = 0; i < size; i++, addr++)
+	for (i = 0; i < size; i++, addr++) {
+		if (i == 0)
+			printf("ssi copy\n");
+
+		if ((i % 10000) == 0)
+			printf("%d ", i);
+
 		dst[i] = dpu_ssi_flash_read(addr);
-	boot_entry();
+
+		if (i == 0)
+			printf("\nfirst coppied\n");
+	}
+
+	printf("copied from ssi\n");
 }
 
 void dpu_ssi_flash_boot(void *boot, uint32_t addr, uint32_t size)
 {
 	dpu_boot_cb boot_func;
 #ifdef CONFIG_DPU_BOOT_STACK
-	__align(4) uint8_t boot_from_stack[512];
+	__align(4) uint8_t boot_from_stack[1024];
 
 	boot_func = (dpu_boot_cb)boot_from_stack;
 	memcpy(boot_from_stack, __dpu_ssi_flash_boot,
-	       sizeof (boot_from_stack));
+	       sizeof(boot_from_stack));
 #else
 	boot_func = __dpu_ssi_flash_boot;
 #endif
 	boot_func(boot, addr, size);
-	unreachable();
+	//unreachable();
```

Comment:
Is this commented for configuring FSBL as a runtime firmware?
If so, please put this line to a single patch, and let's discuss it there.